mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -284,6 +285,59 @@ class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib GaussianMixtureEM.train()
+   * Returns a list containing weights, mean and covariance of each mixture component.
+   */
+  def trainGaussianMixtureEM(
+      data: JavaRDD[Vector], 
+      k: Int, 
+      convergenceTol: Double, 
+      seed: Long, 
+      maxIterations: Int): JList[Object] = {
+    val gmmAlg = new GaussianMixtureEM()
+      .setK(k)
+      .setConvergenceTol(convergenceTol)
+      .setSeed(seed)
+      .setMaxIterations(maxIterations)
+    try {
+      val model = gmmAlg.run(data.rdd.persist(StorageLevel.MEMORY_AND_DISK))
+      var wtArray:Array[Double] = Array()
```

Comment:
Instead of passing mu & sigma as arrays, I tried to directly pass "gaussians "(Array[MultivariateGaussian]). But I was not able to access the attributes of the MultivariateGaussian class object in python. Then I converted "gaussians" to 2 arrays of mu and sigma and returned to python. Which method is good? And is it possible to access the attributes mu & sigma in python by passing "gaussians" directly?